plat/kvm/x86/multiboot.S

```diff
@@ -49,11 +49,11 @@ multiboot_header:
 	.long MULTIBOOT_HEADER_MAGIC
 	.long MULTIBOOT_FLAGS
 	.long -(MULTIBOOT_HEADER_MAGIC+MULTIBOOT_FLAGS) /* checksum */
-	.long multiboot_header				/* header addr */
-	.long 0x100000					/* load addr */
-	.long __bss_start				/* load end addr */
-	.long _end					/* bss end addr */
-	.long multiboot_start32				/* entry addr */
+	.long 0						/* header addr */
+	.long 0						/* load addr */
+	.long 0						/* load end addr */
+	.long 0						/* bss end addr */
+	.long 0						/* entry addr */
```

Comment:
These fields can be removed. If `AOUT_KLUDGE` is not defined the fields do not need to be present.